hieradata/dev/wso2/wso2greg/5.2.0/default/publisher.yaml

```diff
@@ -0,0 +1,159 @@
+# Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+classes:
+  - wso2greg
+wso2::pack_filename: "%{::product_name}-%{::product_version}.zip"
```

Comment:
Please refer hiera.yaml file. Common values like these are retrieved from the default.yaml file.